lib/src/models/parsed_signing_request.dart

```diff
@@ -13,6 +13,6 @@ sealed class ParsedSigningRequest with _$ParsedSigningRequest {
     required ParsedTransaction tx,
     required TransactionSigningModes signingMode,
     required List<List<int>> additionalWitnessPaths,
-    required ParsedTransactionOptions options,
+    ParsedTransactionOptions? options,
```

Comment:
A bit confused with the this and its purpose. In ts, was this provided directly in Transaction? Or its value was induced and only sent to "ParsedTransaction"?

Reply:
In ts, the options come from the SignTransactionRequest, are parsed into ParsedTransactionOptions, and then included in the ParsedSigningRequest.

ts code :
`function parseTxOptions(
  options: TransactionOptions | undefined,  
): ParsedTransactionOptions {
  return {
    tagCborSets: options?.tagCborSets || false,
  }
}`